setup.py

```diff
@@ -8,13 +8,11 @@
     install_requires=[
         # client:
         "requests",
-        "urllib3<2.0.0, >=1.26.0",
+        "urllib3>=1.26.0",
         "pydantic>=2.0.0",
-        "typing-extensions>=4.5.0",
```

Comment:
this dep is only introduced to use @deprecated annotation. Since we do not need this dep now, I'll remove this restraint.